src/hooks/useStore.js

```diff
@@ -214,6 +216,43 @@ const useStore = create(
 				})
 			);
 		},
+		stateToFeatureView: () => {
+			set(
+				produce(draft => {
+					draft.appState = 'featureview';
+				})
+			);
+		},
+		//End AppState Functions
+		//FeatureCardSmall Functions
+		selectFeature: (index1, index2) => {
```

Comment:
instead of index1, index2 try a naming like this: "projectIndex", "featureIndex" and "todoIndex"